client/signup/steps/store-features/index.tsx

```diff
@@ -157,8 +163,14 @@ export default function StoreFeaturesStep( props: Props ): React.ReactNode {
 				page.redirect( `/start/woocommerce-install/?site=${ siteSlug }` );
 				break;
 
-			case 'simple':
-				throw new Error( 'Not yet implemented' );
+			case 'simple': {
+				if ( blockEditorSettings?.is_fse_eligible ) {
```

Comment:
Note that `is_fse_eligible` will tell you that the site is stickered for FSE, but not anything about the theme working with FSE or not, or if the Site editor option is toggled on.

`is_fse_active` is probably a better check here, as this will tell you the theme is compatible with FSE and the Site editor is enabled.

Reply:
Thanks Grant, I've updated this!